webui/react/src/services/decoder.ts

```diff
@@ -90,10 +90,22 @@ export const mapV1MasterInfo = (data: Sdk.V1GetMasterResponse): DeterminedInfo =
     return acc;
   }, BrandingType.Determined);
 
+  // modifiedAt: new Date(data.modifiedAt || 1).getTime(),
+  // TODO: time types?
+  const clustMsg = data.clusterMessage
+    ? ({
+        createdTime: data.clusterMessage.createdTime,
+        endTime: data.clusterMessage.endTime ? new Date(data.clusterMessage.endTime) : undefined,
+        message: data.clusterMessage.message,
+        startTime: data.clusterMessage.startTime,
+      } as ClusterMessage)
```

Comment:
can we avoid using `as`?

Reply:
Good call! Figured out why I was getting "cannot assign" errors.